File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteStop.java
package com.alvinhkh.buseta.kmb.model;

import com.google.gson.annotations.SerializedName;

public class KmbRouteStop {

  @SerializedName("CName")
  public String nameTc;

  @SerializedName("Y")
  public String Y;

  @SerializedName("ELocation")
  public String locationEn;

  @SerializedName("X")
  public String X;

  @SerializedName("AirFare")
  public String airFare;

  @SerializedName("EName")
  public String nameEn;

  @SerializedName("SCName")
  public String nameSc;

  @SerializedName("ServiceType")
  public String serviceType;

  @SerializedName("CLocation")
  public String locationTc;

  @SerializedName("BSICode")
  public String bsiCode;

  @SerializedName("Seq")
  public String seq;

  @SerializedName("SCLocation")
  public String locationSc;

  @SerializedName("Direction")
  public String direction;

  @SerializedName("Bound")
  public String bound;

  @SerializedName("Route")
  public String route;
}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/KmbRouteStop.kt
package com.alvinhkh.buseta.kmb.model

import com.google.gson.annotations.SerializedName

data class KmbRouteStop(
        @SerializedName("CName")
        var nameTc: String = "",

        @SerializedName("Y")
        var Y: String = "",

        @SerializedName("ELocation")
        var locationEn: String = "",

        @SerializedName("X")
        var X: String = "",

        @SerializedName("AirFare")
        var airFare: String = "",

        @SerializedName("EName")
        var nameEn: String = "",

        @SerializedName("SCName")
        var nameSc: String = "",

        @SerializedName("ServiceType")
        var serviceType: String = "",

        @SerializedName("CLocation")
        var locationTc: String = "",

        @SerializedName("BSICode")
        var bsiCode: String = "",

        @SerializedName("Seq")
        var seq: String = "",

        @SerializedName("SCLocation")
        var locationSc: String = "",

        @SerializedName("Direction")
        var direction: String = "",

        @SerializedName("Bound")
        var bound: String = "",

        @SerializedName("Route")
        var route: String = ""
)